model/registry.go

```diff
@@ -15,12 +15,28 @@
 package model
 
 import (
+	"sort"
+
 	"github.com/golang/glog"
 	"github.com/golang/protobuf/proto"
 
 	proxyconfig "istio.io/manager/model/proxy/alphav1/config"
 )
 
+// The Registry describes a set of platform agnostic APIs that must be
+// supported by the underlying platform to store and retrieve routing
+// rules. The code in proxy/* uses these interfaces to retrieve the routing
```

Comment:
We'll reorganize the source tree. Leaving code references like this is not good.